pkg/engine/lifeycletest/refresh_test.go

```diff
@@ -382,6 +382,19 @@ func validateRefreshDeleteCombination(t *testing.T, names []string, targets []st
 
 	provURN := p.NewProviderURN("pkgA", "default", "")
 
+	// The new resources will of had their default provider urn filled in. We fill this in on
```

Comment:
"resources will of had" <- typo

Reply:
Sorry, I don't see what the typo is. "will have had"? "would of had"?

Reply:
Possibly.. I'm not sure what you're trying to say. It seems to me the `of` word is misplaced. But I'm not a native English speaker so it's mostly up to you, just calling attention to a possible typo.

Reply:
Yes I think this is the standard native english speaker thinks/hears "will've" which sounds like "will of" but actually its "will have".